modules/ROOT/examples/live-demos/advtemplate-readonly-template/example.js

```diff
@@ -0,0 +1,134 @@
+const headers = {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json',
+  }
+  
+  const handleResponse = (message) => (response) => {
+    if (!response.ok) {
+      response.text().then((message) => console.error(message))
+      throw new Error(message);
+    } else {
+      return response.json();
+    }
+  }
+  
+  const advtemplate_list = () => {
+    return fetch('/categories', {
+        method: 'GET',
+        headers,
+      })
+      .then(handleResponse('Failed to get template list'))
+  }
+  
+  const advtemplate_get_template = (id) => {
+    return fetch('/templates/' + id, {
+        method: 'GET',
+        headers,
+      })
+      .then(handleResponse('Failed to get template'))
+  }
```

Comment:
```suggestion
  const advtemplate_get_template = (id) => fetch('/templates/' + id, {
      method: 'GET',
      headers,
    })
    .then(handleResponse('Failed to get template'))
```
Simplify the callback by using a shorthand for return